### DISCUSSION TITLE
Search Assets Issue

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#9179] [Q&A / Support]
Hi,
I am facing issue with assets search, I have a check on pimcore demo website and I found the issue there also.
I have added image element and going for add image from Assets so I have click on the search icon and I didn't found any images on list, also I have click on reload button and then it's return error, If I click on search button then it will show images only. ==> [https://demo.pimcore.fun/admin]
![image](https://user-images.githubusercontent.com/52573912/119088831-ea751f80-ba26-11eb-8c33-f49c53bf65f4.png)

Also, in my project, I click on the search button it was return a response like 
![image](https://user-images.githubusercontent.com/52573912/119089166-58b9e200-ba27-11eb-855e-ba3e4a37e7e1.png)
but I can not able to see the image list, If I type something on the search box and click on the search button then I was able to see the list of images on pimcore v6.8.4.

Is there any input or solusion.

**Comment by weisswurstkanone** [comment]
Could you please provide the full stack trace (just click on "Details") thx

**Reply by ghost** [reply]
Hi @weisswurstkanone,

Timestamp: Tue May 25 2021 17:54:47 GMT+0530 (India Standard Time)
Status: 500 | Internal Server Error
URL: /admin/search/search/find?_dc=1621945486162
Method: GET
Message: Argument 1 passed to Pimcore\Bundle\AdminBundle\Controller\Searchadmin\SearchController::filterQueryParam() must be of the type string, null given, called in E:\xampp\htdocs\pimcore_project\vendor\pimcore\pimcore\bundles\AdminBundle\Controller\Searchadmin\SearchController.php on line 68
Trace: 
in E:\xampp\htdocs\pimcore_project\vendor\pimcore\pimcore\bundles\AdminBundle\Controller\Searchadmin\SearchController.php:399
#0 E:\xampp\htdocs\pimcore_project\vendor\pimcore\pimcore\bundles\AdminBundle\Controller\Searchadmin\SearchController.php(68): Pimcore\Bundle\AdminBundle\Controller\Searchadmin\SearchController->filterQueryParam(NULL)
#1 E:\xampp\htdocs\pimcore_project\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(158): Pimcore\Bundle\AdminBundle\Controller\Searchadmin\SearchController->findAction(Object(Symfony\Component\HttpFoundation\Request), Object(Symfony\Component\HttpKernel\Debug\TraceableEventDispatcher), Object(Pimcore\Bundle\AdminBundle\Helper\GridHelperService))
#2 E:\xampp\htdocs\pimcore_project\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\HttpKernel.php(80): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1)
#3 E:\xampp\htdocs\pimcore_project\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(201): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true)
#4 E:\xampp\htdocs\pimcore_project\web\app.php(35): Symfony\Component\HttpKernel\Kernel->handle(Object(Symfony\Component\HttpFoundation\Request))
#5 {main}

**Comment by weisswurstkanone** [comment]
Unable to reproduce this on 6.8.x-dev
